options.js

```diff
@@ -187,7 +187,7 @@ function importCookies(files) {
                     lines = []; // Skip line parsing, this will also make lines.length == 0 for the finish check
                 }
                 for (let a = 0; a < lines.length; a++) {
-                    const l = lines[a];
+                    const l = lines[a].replace('#HttpOnly_','');
```

Comment:
To set the HttpOnly property we could do something like this:
```suggestion
                    const httpOnly = lines[a].startsWith('#HttpOnly_');
                    const l = lines[a].replace('#HttpOnly_','');
```

Then at the bottom we add it to the `cookie_obj`:
```javascript
const cookie_obj = {
    ...,
    'httpOnly': httpOnly
};
```

Reply:
Indeed that'd be correct, in my case I didn't care much for the security aspect admittedly :)